lib/dialects/sqlite3/query/sqlite-querycompiler.js

```diff
@@ -82,62 +84,34 @@ class QueryCompiler_SQLite3 extends QueryCompiler {
       });
     }
 
-    if (insertData.values.length === 1) {
-      const parameters = this.client.parameterize(
-        insertData.values[0],
-        this.client.valueForUndefined,
-        this.builder,
-        this.bindingsHolder
-      );
-      sql += ` values (${parameters})`;
-
-      const { onConflict, ignore, merge } = this.single;
-      if (onConflict && ignore) sql += this._ignore(onConflict);
-      else if (onConflict && merge) {
-        sql += this._merge(merge.updates, onConflict, insertValues);
-        const wheres = this.where();
-        if (wheres) sql += ` ${wheres}`;
-      }
-
-      const { returning } = this.single;
-      if (returning) {
-        sql += this._returning(returning);
-      }
-
-      return {
-        sql,
-        returning,
-      };
+    if (
+      !this.client._satisfiesVersion([3, 7, 11]) &&
```

Comment:
I'd rather do this check once on driver instantiation and set a feature boolean flag for subsewuent chrcks

Reply:
@kibertoad what do you want to do if we initialize with the driver with a version below the 3.7.11 threshold? Throw an error and halt, forcing at least that version? Something else?

Reply:
Wouldn't it break anyway if someone attempts a multi-value JSON insert anyway? Do we need to force-break anything?